src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/Transaction.java

```diff
@@ -0,0 +1,75 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+import java.util.Optional;
+
+public class Transaction extends Section {
+    @Getter(lazy = true)
+    private final ReferenceTransactionNumber referenceTransactionNumber =
+        ReferenceTransactionNumber.fromString(extractSegment(ReferenceTransactionNumber.KEY_QUALIFIER));
+
+    /*
+    @Getter(lazy = true)
```

Comment:
I don't see any benefit in committing this commented out code. We can always add it later, if and when we need it

Reply:
I'm in favour of leaving them commented out until [NIAD-873](https://gpitbjss.atlassian.net/browse/NIAD-873) but if we're all in agreement, I'm happy to remove them @andrewwa-kainos @n-r-b

Reply:
Removed the commented out code